frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -12,7 +13,10 @@ import { ServiceModel } from '../models';
 import { getKnativeEventSourceIcon } from './get-knative-icon';
 import { getEventSourceModels } from './fetch-dynamic-eventsources-utils';
 
-export const getEventSourcesDepResource = (formData: EventSourceFormData): K8sResourceKind => {
+export const isKnownEventSource = (eventSource: string): boolean =>
+  !!Object.keys(EventSources).includes(eventSource);
+
+export const getEventSourcesDepResource = (formData: any): K8sResourceKind => {
```

Comment:
why changed type from `EventSourceFormData` to `any`

```suggestion
export const getEventSourcesDepResource = (formData: EventSourceFormData): K8sResourceKind => {
```

Reply:
changed back to `EventSourceFormData` and fixed the type error arising between `EventSourceFormData` and `FormikValues`